geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseConnectionPool.scala

```diff
@@ -114,7 +114,7 @@ object HBaseConnectionPool extends LazyLogging {
       override def run(): Connection = {
         if (validate) {
           logger.debug("Checking configuration availability")
-          HBaseAdmin.checkHBaseAvailable(conf)
+          HBaseAdmin.available(conf)
```

Comment:
we should move this to `org.locationtech.geomesa.hbase.utils.HBaseVersions` and add the methods from 1.4 so that it will work with both versions.